### ISSUE TITLE
Empty response when expanding account_tax_ids for invoice

### ISSUE THREAD

**Issue by jasonlewis** [stripe/stripe-mock#445]
When trying to use the retrieve endpoint for an invoice with `account_tax_ids` expanded the mock server returns an empty response.

```bash
$ curl -vvvv -g "http://localhost:12111/v1/invoices/in_1NhTSDJ7Bu9fw2TI7tY1f5zD?expand[]=account_tax_ids" -u sk_test_123:
*   Trying ::1:12111...
* TCP_NODELAY set
* Connected to localhost (::1) port 12111 (#0)
* Server auth using Basic with user 'sk_test_123'
> GET /v1/invoices/in_1NhTSDJ7Bu9fw2TI7tY1f5zD?expand[]=account_tax_ids HTTP/1.1
> Host: localhost:12111
> Authorization: Basic c2tfdGVzdF8xMjM6
> User-Agent: curl/7.68.0
> Accept: */*
> 
* Empty reply from server
* Connection #0 to host localhost left intact
curl: (52) Empty reply from server
```

**Comment by anniel-stripe** [comment]
Thanks for opening this issue! We're looking into this.

**Comment by anniel-stripe** [comment]
The fix has been released in [v0.171.0](https://github.com/stripe/stripe-mock/releases/tag/v0.171.0). Thanks again for raising this issues!
